generator/config/expression/switch.yaml

```
@@ -24,3 +24,47 @@ arguments:
         description: |
             The path to take if no branch case expression evaluates to true.
             Although optional, if default is unspecified and no branch case evaluates to true, $switch returns an error.
+tests:
+    -
+        name: 'Example'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/switch/'
+        pipeline:
+            -
+                $project:
+                    name: 1
+                    summary:
+                        $switch:
+                            branches:
+                                -
+                                    case:
+                                        $gte:
+                                            -
+                                                $avg:
+                                                    - '$scores'
+                                            - 90
+                                    then: 'Doing great!'
+                                -
+                                    case:
+                                        $and:
+                                            -
+                                                $gte:
+                                                    -
+                                                        $avg:
+                                                            - '$scores'
```

On the [example](https://www.mongodb.com/docs/manual/reference/operator/aggregation/switch/), this is not an array. But the `$avg` operator accept both a single value or a list of 1, having the same behavior.

```suggestion
                                                        $avg: '$scores'
```